src/main/java/CompanyList.java

```diff
@@ -5,4 +5,16 @@
 public class CompanyList {
 
     ArrayList<Company> companies = new ArrayList<>();
+
+    public void list(){
```

Comment:
Perhaps consider renaming the method `list` to `listCompanyInformation` to differentiate between listing companies and listing venues